src/main/java/com/fundation/webservice/model/ThumbnailImage.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+/**
+ * Implements Thumbnail class an convert Method.
+ *
+ * @author Maday Alcalá Cuba
+ * @version 1.0
+ */
+public class ThumbnailImage {
+    CriteriaThumbnailImage criteria;
+
+    public ThumbnailImage(CriteriaThumbnailImage criteria) {
+        this.criteria = criteria;
+    }
+
+    //The convert method capture a thumbnail of a frame specific.
```

Comment:
Please use the adopted convention for documenting methods.

Reply:
solved